libobs/obs-audio.c

```diff
@@ -508,12 +544,13 @@ bool audio_callback(void *param, uint64_t start_ts_in, uint64_t end_ts_in, uint6
 				continue;
 			if (!obs_source_active(source))
 				continue;
-
-			obs_source_enum_active_tree(source, push_audio_tree, audio);
-			push_audio_tree(NULL, source, audio);
-
+			// first, add top-level sources as root_nodes
```

Comment:
Use C-style comments and use capitals at the beginning of sentences (except if the sentence begins with the name of a variable which obviously would be lower-case).